app/src/main/java/me/ccrama/redditslide/Activities/MultiredditOverview.java

```diff
@@ -211,7 +211,7 @@ public void onClick(@NonNull MaterialDialog materialDialog,
                 return true;
 
             case R.id.subs:
-                ((DrawerLayout) findViewById(R.id.drawer_layout)).openDrawer(Gravity.RIGHT);
+                ((DrawerLayout) findViewById(R.id.drawer_layout)).openDrawer(Gravity.END);

```

Comment:
I changed these from RIGHT to END anywhere they existed. Nothing should change for LTR users but RTL users aren't left in the dust anymore